### PR TITLE
MGDCTRS-774: Chore: Add the alert for expired namespace and KI in dup…

### PR DESCRIPTION
**Pull request by indraraj**
Issue : https://issues.redhat.com/browse/MGDCTRS-774

Update the Alert for Namespace `expired | Deleting` state.
image
updated the Alert content in duplicate flow for the connector selection step.
image

Added the alert if earlier selected KI or Namespace is no longer available in duplicate flow.
image
image

### CODE REVIEW

File: src/app/pages/CreateConnectorPage/StepNamespace.tsx [hunk 1]
```diff
@@ -104,6 +107,15 @@ const ClustersGallery: FunctionComponent = () => {
     onRefresh();
   };
 
+  const getEvalNamespaceAlert = (expiration: string): string => {
+    const { hours, min } = getPendingTime(new Date(expiration));
+    console.log(hours, min);
```
Comment (gashcrumb) [inline src/app/pages/CreateConnectorPage/StepNamespace.tsx:112]: It'd be good to comment this out or remove unless you expect this to be troublesome.

Comment (indraraj) [reply src/app/pages/CreateConnectorPage/StepNamespace.tsx:112]: yes, this was not intentional I will remove it

Comment (indraraj) [reply src/app/pages/CreateConnectorPage/StepNamespace.tsx]: done

**Review by gashcrumb — APPROVED** [review]
:+1:  looks great!

File: src/app/pages/CreateConnectorPage/StepNamespace.tsx (hunk 1 above)
Comment (gashcrumb) [reply src/app/pages/CreateConnectorPage/StepNamespace.tsx]: no worries, I forget them all the time :-)

**Review by uidoyen — APPROVED** [review]
LGTM!

**Pull request merged**